MockitoMockMaker provides mocking of static methods (#1756)
spockframework/spock: docs/extensions.adoc

please see the Mockito manual "Mocking final types, enums and final methods" for all pros and cons,
when using `org.mockito.MockMakers.INLINE`.

> The `mockito` Mock Maker also supports mocking of static methods of classes and interfaces with `Mock/Stub/SpyStatic`.

Author: Done

please see the Mockito manual "Mocking final types, enums and final methods" for all pros and cons,
when using `org.mockito.MockMakers.INLINE`.

> 
> 
> The `mockito` mock maker also supports mocking of static methods of classes and interfaces with `Mock/Stub/SpyStatic`.
> See <<interaction_based_testing.adoc#static-mocks,static mocks>> section for more details.

Reviewer: ```suggestion
It also supports mocking of static methods of classes and interfaces with `Mock/Stub/SpyStatic`.
See <<interaction_based_testing.adoc#static-mocks,static mocks>> section for more details.
```
